### DISCUSSION TITLE
support camelCase without boilerplate

### DISCUSSION THREAD

**Discussion by uriva** [fastapi/fastapi#7834] [Questions]
Sorry if this is already possible (I couldn't figure out how to do it) -

I want to have a request class with python style snake case naming, but still send it from js/nodejs in camelcase, and have the parsing work ootb.

A flag in the `Config` would be convenient, maybe with this lib: https://github.com/nficano/humps

**Comment by uriva** [comment]
actually I now understand that this is more related to pydantic than fastapi.

**Comment by tiangolo** [comment]
Thanks for coming back to close the issue :+1:
